release_notes/ocp-4-13-release-notes.adoc

```diff
@@ -3955,3 +3955,32 @@ $ oc adm release info 4.13.36 --pullspecs
 [id="ocp-4-13-36-updating"]
 ==== Updating
 To update an existing {product-title} 4.13 cluster to this latest release, see xref:../updating/updating-cluster-cli.adoc#updating-cluster-cli[Updating a cluster using the CLI].
+
+[id="ocp-4-13-37"]
+=== RHBA-2024:1200 - {product-title} 4.13.37 bug fix
+
+Issued: 2024-03-13
+
+{product-title} release 4.13.37 is now available. The list of bug fixes that are included in the update is documented in the link:https://access.redhat.com/errata/RHBA-2024:1200[RHBA-2024:1200] advisory.
+
+Space precluded documenting all of the container images for this release in the advisory.
+
+You can view the container images in this release by running the following command:
+
+[source,terminal]
+----
+$ oc adm release info 4.13.37 --pullspecs
+----
+
+[id="ocp-4-13-37-bug-fixes"]
+==== Bug fixes
+
+* Previously, the `manila-csi-driver-controller-metrics` service had empty endpoints because an incorrect name for the app selector. With this release the app selector name is changed to `openstack-manila-csi` and the issue is fixed.(link:https://issues.redhat.com/browse/OCPBUGS-26224[*OCPBUGS-26224*])
+
+* Previously, the `ovn-ipsec-containerized` and the `ovn-ipsec-host` daemons contained a typographical error for a `openssl` parameter: `-checkedn` instead of `checkend`. This error caused certificate rotation to occur after every `ovn-ipsec` pod restart. Now, the parameter name is fixed, so that the certificate that the Internet Protocol Security (IPsec) automatically rotates as expected. (link:https://issues.redhat.com/browse/OCPBUGS-30150[*OCPBUGS-30150*])
```

Comment:
Something about the last sentence was reading strange to me. Not sure if it's a missing word. Something like this?
```suggestion
* Previously, the `ovn-ipsec-containerized` and the `ovn-ipsec-host` daemons contained a typographical error for a `openssl` parameter: `-checkedn` instead of `checkend`. This error caused certificate rotation to occur after every `ovn-ipsec` pod restart. With this release, the parameter name is fixed, and the certificate used by the Internet Protocol Security (IPsec) now automatically rotates as expected. (link:https://issues.redhat.com/browse/OCPBUGS-30150[*OCPBUGS-30150*])
```